lib/jwt_signed_request/verify.rb

```diff
@@ -92,5 +93,25 @@ def verified_headers?
         Headers.fetch(header_key, request) == header_value
       end
     end
+
+    def parsed_claims_path
```

Comment:
There's a lot of `path` references that are slightly different. Would it be clearer to refer to them using `_uri` / `uri` suffixes? e.g.

```
def standard_query_values(uri)
  URI.decode_www_form(uri.query)
```

def parsed_claims_uri
```